utils/lib/rr_graph/cyclic_to_acyclic.py

```diff
@@ -0,0 +1,201 @@
+class CycleSeas:
```

Comment:
CycleSeas?

Does this terminology come from somewhere?

Reply:
Nope.  I had a crack at googling what the appropriate term was in graph theory, but didn't find anything.  So I made one up.

Reply:
Correct term is : Strongly Connected Subgraph